src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotJVMCIRuntime.java

```diff
@@ -192,15 +192,6 @@ public static HotSpotJVMCIRuntime runtime() {
                     // initialized.
                     JVMCI.getRuntime();
                 }
-                // Make sure all the primitive box caches are populated (required to properly
```

Comment:
This code has not been necessary ever since https://github.com/openjdk/jdk/blob/e56bcb04b11c6494e6afdf0bd9b9bc65a4769347/src/hotspot/share/jvmci/jvmciCodeInstaller.cpp#L985 was added.

Reply:
How are the caches uninitialized if we have forced their initialization?

Reply:
A box cache initialization can fail due to `StackOverflowError`. In this case, we must not rematerialize a virtual object corresponding to a value that would be read from a box cache.